channels/cl.m3u

```diff
@@ -99,8 +99,6 @@ https://593b04c4c5670.streamlock.net/8038/8038/playlist.m3u8
 https://stv.janus.cl/playlist/stream.m3u8
 #EXTINF:-1 tvg-id="Teletrak.cl" tvg-name="Teletrak" tvg-country="CL" tvg-language="Spanish" tvg-logo="https://i.imgur.com/6J8xRah.png" group-title="",Teletrak (720p)
 https://unlimited6-cl.dps.live/teletrak/teletrak.smil/playlist.m3u8
-#EXTINF:-1 tvg-id="Teletrak.cl" tvg-name="Teletrak" tvg-country="CL" tvg-language="Spanish" tvg-logo="https://i.imgur.com/6J8xRah.png" group-title="",Teletrak (720p) [Offline]
-https://unlimited1-us.dps.live/teletrak/teletrak.smil/playlist.m3u8
```

Comment:
Not 24/7

![image](https://user-images.githubusercontent.com/30985701/132142109-44e10e6d-263e-4dc0-8214-15cfb0727007.png)